test_tsp.py

```diff
@@ -245,6 +249,36 @@ def test_fetch_xy_tree(self, kernel):
         self._delete_experiments()
         self._delete_traces()
 
+    def test_fetch_xy(self, kernel):
+        traces = []
+        response = self.tsp_client.open_trace(os.path.basename(kernel), kernel)
+        traces.append(response.model.UUID)
+        response = self.tsp_client.open_experiment(
+            os.path.basename(kernel), traces)
+        assert response.status_code == 200
+        experiment_uuid = response.model.UUID
+
+        response = self.tsp_client.fetch_experiment_outputs(experiment_uuid)
+        output_id = response.model.descriptors[0].id
+        status = ResponseStatus.RUNNING.name
+        while status == ResponseStatus.RUNNING.name:
+            time.sleep(1)
+            response = self.tsp_client.fetch_xy_tree(
+                experiment_uuid, output_id)
+            status = response.model.status.upper()
+
+        requested_items = [1, 2]
```

Comment:
Based on testXYDataProvider in incubator's DataProviderServiceTest, these hardcoded series IDs should be replaced with actual one(s), obtained from the final response above; WIP.

Reply:
-Done; pushed herein, just now.